website/content/partials/x509-sha1-deprecation.mdx

```diff
@@ -0,0 +1,5 @@
+~> **Note**: This engine can use external X.509 certificates or load certificates into Vault.
```

Comment:
Given the wide set of plugins here, which aren't strictly limited to x.509 certificate usage, you're probably best adding a notice here saying not just use/load but also TLS (which is the bigger place this will likely fail).

Reply:
See comment above; in general _loading_ will be fine, its the actual TLS usage that'll fail IIUC.

Reply:
I don't actually know if _outbound_ connections will verify certs also. That might be worth testing and seeing if Go still allows SHA-1 client certs to a non-SHA-1 server.

Reply:
Ah, right. But most of the time we're using the certs for TLS connections, or validating signatures (`aws`, `cert` auth methods), so I think this applies. But I'll clarify that it isn't loading, but signature verification.

Reply:
Yeah, hence my suggestion to include TLS in the mix ;-)